tests/ci/commit_status_helper.py

```diff
@@ -73,3 +74,28 @@ def post_labels(gh, pr_info, labels_names):
     pull_request = repo.get_pull(pr_info.number)
     for label in labels_names:
         pull_request.add_to_labels(label)
+
+
+def fail_simple_check(gh, pr_info, description):
+    if SKIP_SIMPLE_CHECK_LABEL in pr_info.labels:
+        return
+    commit = get_commit(gh, pr_info.sha)
+    commit.create_status(
+        context="Simple Check",
+        description=description,
+        state="failed",
```

Comment:
https://docs.github.com/en/rest/commits/statuses#about-the-commit-statuses-api

It should be `failure`